bin/deploy.sh

```diff
@@ -1,21 +0,0 @@
-#! /bin/bash
```

Comment:
Why you remove this file?

Reply:
I thot, that bin order not necessary here. Are we needed it here?

Reply:
You don't want to deploy application to staging server? Why you change your mind?)
https://github.com/Social-projects-Rivne/Lunch_Time/pull/45

Reply:
Ok, it will be returned